app/controllers/application_controller/timelines.rb

```diff
@@ -102,42 +102,54 @@ def tl_build_timeline_report_options
       end
 
       temp_clause = @tl_record.event_where_clause(@tl_options.evt_type)
+      conditions = [temp_clause[0], "timestamp >= ?", "timestamp <= ?"]
+      parameters = temp_clause[1..-1] + [from_dt, to_dt]
```

Comment:
Minor, but instead of temp_clause[0] and temp_clause[1], for readability let's give them names, and set them with decomposition, so maybe...

```ruby
conditions, *parameters = @tl_record.event_where_clause(@tl_options.evt_type)
conditions += ["timestamp >= ?", "timestamp <= ?"]
parameters += [from_dt, to_dt]
```